core/azure-core/src/main/java/com/azure/core/util/logging/ClientLoggerAPI.java

```diff
@@ -1,12 +1,12 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 
-package com.azure.core.implementation.logging;
+package com.azure.core.util.logging;
 
 /**
  * Fluent logging API interface.
  */
-public interface ServiceLoggerAPI {
+public interface ClientLoggerAPI {
```

Comment:
What's the naming convention for interfaces? I see that none of the other interfaces like `Page`, `PagedResponse`, `Response`, `Tracer` etc don't have API suffix. So, would it be better to keep it consistent and just name this as `ClientLogger` or simply `Logger`

Reply:
I agree that this should just be called Logger, however I don't know that this interface needs to be public either - it can probably be package-private, and the ClientLogger changed to just return the concrete ClientLogger class, rather than the interface.

Reply:
I also do not see need for interface in this case. I prefer removing  ClientLoggerAPI ?

Reply:
That is fine, as long as the no-op logger continues to function.

Reply:
The whole interface, and likely the no-op logger, could go given how the ClientLogger is now structured to early out if logging isn't supported at a specific level. It was different at another point in time which is why the interface was in place.

Reply:
removed ClientLoggerAPI